### ISSUE TITLE
[a11y] Explicitly set the default language of the page to English (e.g. from `<html>` to `<html lang="en">`)

### ISSUE THREAD

**Issue by fititnt** [SimonbJohnson/quickX3#73]
Refs:
- https://www.w3.org/International/questions/qa-html-language-declarations
- https://achecker.ca/checker/suggestion.php?id=48
  - Can do the test again using https://achecker.ca/checker/
- https://webaim.org/standards/wcag/checklist#sc3.1.1
  - See https://wave.webaim.org/report#/hxldash.com
- https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2
- **Potential specification to NOT set English as default** 
  - https://www.w3.org/TR/i18n-html-tech-lang/#h2_overall
    - "Always declare the default language for text in the page using attributes on the html tag, unless the document contains content aimed at speakers of more than one language. more..."

---

The default behavior of the hxldash at this moment is not explicitly set the language of the document, and this result in failing an accessibility test. **My suggestion at the moment is explicitly change the `<html>` to `<html lang="en">` on the entire site (even on the dashboards page) that may have mixed content to pass these accessibility tests.** (Maybe is at this line https://github.com/SimonbJohnson/quickX3/blob/master/hxldash/templates/base/base.html#L5)

I will open other 1~2 issues on some corner cases. But this issue is likely to be a quick one.

---

### Why
By not explicitly setting the language, screen reader users that by default use other languages (even if they understand English) will hear their screen readers speak English with an different voice.

One way to simulate this is going to google translate page, putting something in English but asking random languages to speak aloud.

**Comment by SimonbJohnson** [comment]
Fixed https://github.com/SimonbJohnson/quickX3/pull/77
